modules/lang-expression/src/test/java/org/elasticsearch/script/expression/ExpressionTermsSetQueryTests.java

```diff
@@ -64,7 +60,8 @@ public void setUp() throws Exception {
         when(fieldData.load(anyObject())).thenReturn(atomicFieldData);
 
         service = new ExpressionScriptEngine();
-        lookup = new SearchLookup(mapperService, (ignored, lookup) -> fieldData);
+        lookup = new SearchLookup(field -> field.equals("field") || field.equals("alias") ? fieldType : null,
+            (ignored, lookup) -> fieldData);
```

Comment:
I wonder what these were testing before. You change doesn't make it worse though!

I wonder if this change in general could make field aliases simpler. Like, in a follow up.

Reply:
I wondered the same, especially why expression tests need to specifically test field aliases, that should not be necessary?

Reply:
I'm guessing that aliases at some point were different beasts and the test tested something. Or maybe we ended up copy and pasting stuff?